index.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+
+const dbhelpers = require('./helpers');
+
+const server = express();
+const PORT = 4000;
```

Comment:
Your code is clean so far. I like that you're including your settings at the top of the file.